witchcraft/refreshable/refreshable_file.go

```diff
@@ -35,73 +34,58 @@ type fileRefreshable struct {
 }
 
 // NewFileRefreshable returns a new Refreshable whose current value is the bytes of the file at the provided path.
+// The duration configures how often the refreshable check the file
 // Calling this function also starts a goroutine which updates the value of the refreshable whenever the specified file
 // is changed. The goroutine will terminate when the provided context is done or when the returned cancel function is
 // called.
-func NewFileRefreshable(ctx context.Context, filePath string) (r Refreshable, rErr error) {
+func NewFileRefreshable(ctx context.Context, filePath string, duration time.Duration) (Refreshable, error) {
```

Comment:
Would prefer not to break this public API. Can we leave the arguments the same and use a default refresh duration (per Brad's suggestion, 1s to match Java sounds fine) and then add another `New` function that specifies the duration?

Reply:
Updated! Sorry didn't realize the FileRefreshable was considered public as well